boards/xtensa/odroid_go/Kconfig.defconfig

```diff
@@ -9,3 +9,13 @@ config BOARD
 
 config ENTROPY_ESP32_RNG
 	default y if ENTROPY_GENERATOR
+
+if DISPLAY
+
+config SPI
+	default y
+
+config ILI9341
+	default y
```

Comment:
```suggestion
config ILI9341
	default DISPLAY
```

Reply:
btw, opened https://github.com/zephyrproject-rtos/zephyr/pull/44116 so that this becomes redundant